backend/backend-standard-and-best-practices.md

```diff
@@ -213,7 +214,19 @@ Most commonly used are:
 
 You can read more about this and other status codes in [this link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-### 6- Routes
+## 6- API Documentation
+
+There are lots of different tools in the market to document APIs. Here in Wolox we decided to use Openapi 3 (A.K.A Swagger) as a standard documentation tool in all of our techs. The reasons behind this decision were some of the advantages OpenApi prodvides:
+
+* **Quick to use**: The properties that you need to complete in every endpoint are always the same and there are only a few. You can also frequently reuse components and the generic information is simple to complete.
```

Comment:
you need to complete -> you  need to fill
simple to complete -> simple to provide